examples/fic-ecl-connection/README.md

```diff
@@ -0,0 +1,9 @@
+## Create Router and ECL-connection
+
+This example provides the FIC router and the connection settings to the ECL.
+
+The default configuration is to do source NAT from "group1" to "group2". This will be used for future connections to Wasabi Object Storage and has no significance at this time.
+ECL connections will be made to "group1".
+Be very careful with the values you set for IP addresses, etc., as they can affect your existing environment.
+
+After setting the variables according to your environment, run `terraform plan`, and if everything is ok, run `terraform apply`.
```

Comment:
ExampleのREADMEはオフィシャルプロバイダー(AWSなど)と同等のクオリティにしたいです。
AWSのExampleのREADMEは以下のような構成になっています。

- xxxxx Example(Exampleの説明)
- Run the example(Exampleの実行方法)
- Remove the example(Exampleで作成したリソースの削除方法)

下のリンク先のTerraform AWSのExampleを参考にしてREADMEの構成をAWSと合わせていただけませんでしょうか？変更例をsuggestionさせていただいておりますのでよろしければそちらも参考にしてください。
https://github.com/hashicorp/terraform-provider-aws/tree/main/examples/alexa


```suggestion

## Create Router and ECL-connection

This example provides the FIC router and the connection settings to the ECL.

The default configuration is to do source NAT from "group1" to "group2". This will be used for future connections to Wasabi Object Storage and has no significance at this time.
ECL connections will be made to "group1".
Be very careful with the values you set for IP addresses, etc., as they can affect your existing environment.

## Run the example

After setting the variables according to your environment, run `terraform plan`, and if everything is ok, run `terraform apply`.

```bash
terraform init
terraform plan
terraform apply
 ```

## Remove the example

```bash
terraform destroy
```

```